circle.yml

```diff
@@ -131,6 +131,8 @@ jobs:
       - run: yarn run build-min
       - run: yarn run build-dev
       - run: yarn run build-css
+      - run: yarn run build-style-spec
+      - run: yarn run test-build
```

Comment:
@jfirebaugh do you see any reason to do `test-build` as a separate workflow step?  I thought it kinda made sense to keep it here, but 🤷‍♂️

Reply:
No, adding it here is what I would have done too.